src/BaseView_TEST.cc

```diff
@@ -152,30 +155,27 @@ TEST_F(BaseViewTest, AddEntities)
       modelNameView.NewEntities().end());
 
   auto e1ConstData = modelNameView.EntityComponentConstData(e1);
-  EXPECT_EQ(e1, std::get<Entity>(e1ConstData));
-  EXPECT_EQ(&e1ModelComp, std::get<const components::Model *>(e1ConstData));
-  EXPECT_EQ(&e1NameComp, std::get<const components::Name *>(e1ConstData));
+  ASSERT_EQ(2u, e1ConstData.size());
+  EXPECT_EQ(&e1ModelComp, e1ConstData[0]);
+  EXPECT_EQ(&e1NameComp, e1ConstData[1]);
 
   auto e1Data = modelNameView.EntityComponentData(e1);
```

Comment:
nit: make sure the vector has 2 elements in it before indexing it to avoid out of bounds access segfault
```suggestion
  auto e1Data = modelNameView.EntityComponentData(e1);
  ASSERT_EQ(2u, e1Data.size());
```